org.mintguide.WebAppReader.yaml

```diff
@@ -0,0 +1,49 @@
+app-id: org.mintguide.WebAppReader
+runtime: org.kde.Platform
+runtime-version: '6.6'
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: '6.6'
+command: /app/webappreader.sh
+finish-args:
+  - --device=dri
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=home
+  - --env=QTWEBENGINEPROCESS_PATH=/app/bin/QtWebEngineProcess
+  - --share=network
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+build-options:
+  env:
+    QTWEBENGINE_DISABLE_SANDBOX: '1'
+modules:
+  - python3-requirements.json
+  - name: webappreader
+    subdir: app
+    buildsystem: simple
+    build-commands:
+      - cp webappreader.py /app/
+      - cp webappreader.sh /app/
+      - chmod +x /app/webappreader.sh
+      - cp constants.py /app/
+      - cp settings.py /app/
+      - cp tools.py /app/
+      - cp tools.py /app/
+      - cp -r ui /app/
+      - cp -r pages /app/
+      - cp -r database /app/
+      - cp -r src /app/
+    sources:
+      - type: dir
+        path: .
+        dest: app
+    post-install:
+      - install -Dm644 org.mintguide.WebAppReader.desktop /app/share/applications/org.mintguide.WebAppReader.desktop
+      - install -Dm644 src/logo.png /app/share/icons/hicolor/128x128/apps/org.mintguide.WebAppReader.png
+      - install -Dm755 wget2 /app/bin/wget2
+      - install -Dm755 libwget.so.2 /app/lib/libwget.so.2
+      - install -Dm755 libgnutls-dane.so.0 /app/lib/libgnutls-dane.so.0
+      - install -Dm755 libunbound.so.8 /app/lib/libunbound.so.8
+      - install -Dm755 libprotobuf-c.so.1 /app/lib/libprotobuf-c.so.1
+      - install -Dm755 libpython3.12.so.1.0 /app/lib/libpython3.12.so.1.0
```

Comment:
You shouldn't copy paste libs like that, build what you need in the manifest, remove things that are in the runtime...

Reply:
these are dependencies for the newer version of wget2 to work

Reply:
why does it require a different python?

Reply:
it first sight you are just doing it wrong.

Reply:
This is the only way my program started normally from runtime, I copied the libraries from fedora39

Reply:
don't do that.
build from source in the manifest.